### PR TITLE
Use HTTP and remove Athenz injection

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

**Comment by hakonhall** [comment]
There were 2 problems:
 1. systemtests failed because AthenzIdentityProvider injection failed, because Athenz doesn't work (isn't supposed to work) in the system tests.
 2. The synthetic config server application was effectively removed on the controller, because I changed the condition for making it from "depend on multitenant" to "depend on hosted vespa", and the controller is multitenant but not hosted (should be the other way around!).

The solution is:
 - Always make synthetic config server application (fixes (2)).
 - Make /state/v1/health available on port 19071 and use that (HTTP) to remove requirement on Athenz. I have kept the support for HTTPS for the lower-level code (HealthMonitor and HealthClient) as adding HTTP support was fairly easy.

Again, this PR should be a no-op.

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/health/ApplicationHealthMonitor.java [hunk 1]
```diff
@@ -6,35 +6,36 @@
...
 /**
  * Responsible for monitoring a whole application using /state/v1/health.
  *
  * @author hakon
  */
 public class ApplicationHealthMonitor implements ServiceStatusProvider, AutoCloseable {
+    public static final String PORT_TAG_STATE = "STATE";
+    public static final String PORT_TAG_HTTP = "HTTP";
+    /** Port tags implying /state/v1/health is served */
+    public static final List<String> PORT_TAGS_HEALTH = Arrays.asList(PORT_TAG_HTTP, PORT_TAG_STATE);
```
Comment (freva) [inline service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/health/ApplicationHealthMonitor.java]: Wrap this in `Collections.unmodifiableList()`?

**Review by freva — APPROVED** [review]

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/health/ApplicationHealthMonitor.java (hunk 1 above)
Comment (hakonhall) [reply service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/health/ApplicationHealthMonitor.java]: Done

**Comment by hakonhall** [comment]
In 6.245.55

**Comment by hakonhall** [comment]
merged during merge-stop, let me revert

**Pull request merged**